scripts/build-windows-provisioning-scripts.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash -e
```

Comment:
It is more common to set bash errors after this line:

```
set -o errexit
set -o nounset
set -o pipefail
```

Reply:
updated